server/libs/auth.js

```diff
@@ -0,0 +1,95 @@
+//
+// SecureImage
+//
+// Copyright © 2018 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Created by Jason Leach on 2018-02-01.
+//
+
+/* eslint-env es6 */
+
+'use strict';
+
+import request from 'request';
+import jwt from 'jsonwebtoken';
+import pemFromModAndExponent from 'rsa-pem-from-mod-exp';
+
+const verifyToken = clientAccessToken => new Promise((resolve, reject) => {
+  request.get('https://dev-sso.pathfinder.gov.bc.ca/auth/realms/mobile/protocol/openid-connect/certs', {
```

Comment:
Lets move this to the config. We can check to see if it should be set in the environment or if its OK for a config file. Either way it should not be hard coded.

Reply:
Alright will work on that